testsuite/openshift/objects/auth_config/sections.py

```diff
@@ -197,27 +197,41 @@ def add_uma(self, name, endpoint, credentials_secret, **common_features):
 
 
 class ResponseSection(Section):
-    """Section which contains response configuration. todo envoyDynamicMetadata"""
+    """Section which contains response configuration."""
 
     @property
-    def success_headers(self):
-        """Nested dict for most of the section."""
-        return self.section.setdefault("success", {}).setdefault("headers", {})
+    def success(self):
```

Comment:
This method is now irrelevant.

Reply:
Removed this property so only `success_headers` and `success_dynamic_metadata` properties are used.